inc/auth.class.php

```diff
@@ -502,7 +502,7 @@ function getAlternateAuthSystemsUserLogin($authtype = 0) {
 
                   $token = User::getToken($cookie_id, 'personal_token');
 
-                  if ($token !== false && Auth::checkPassword($token, $cookie_token)) {
+                  if (Auth::checkPassword($cookie_token, $hash)) {
```

Comment:
in 9.3, it's:
```
if (Auth::checkPassword($cookie_token, $token)) {
```

Reply:
You're right, thank you. I also saw that `personal_token` was still used instead of `cookie_token`.